README.md

```diff
@@ -4,122 +4,145 @@ Kubernetes manifest files for [notification.canada.ca](https://notification.cana
 ```
 
-- the `make encrypt-production` command that will encrypt environment variables in production.
+2. Add a corresponding value in the overrides file in helmfile/overrides/notify/\<component\>.yaml.gotmpl for the component. You have choices on how to configure this. 
 
+If this is a straight up true/false feature flag, use 
 
-```sh
-AWS_PROFILE=notify-production make decrypt-production
-# Change values in the decrypted file at env/production/.env
-# Encrypt the decrypted file that you just edited
-AWS_PROFILE=notify-production make encrypt-production
-# Creates a new file at env/production/.env.enc.aws which is safe to commit
+```yaml
+SOME_NEW_ENV_VAR: "{{ .StateValues.SOME_NEW_ENV_VAR }}"
 ```
 
-## How do I add a new environment variable?
+If this is a string that changes based on environment name, you can leverage built in helmfile keywords:
 
-As mentioned above, you will need to make changes to the `.env` file to include them in the `ConfigMap` object. In addition, you need to set up the `kustomization.yaml` file to include the new environment variable in the `ConfigMap`. This would look something like this if you wanted to add the variable `FOO` with the value `BAR`:
+```yaml
+SOME_NEW_ENV_VAR: "notify-new-env-var-{{ .Environment.Name }}"
+```
 
-In a `.env` file add:
+If this is a URL, you can leverage the BASE_DOMAIN environment variable that is set in getContext.sh
 
+```yaml
+SOME_NEW_ENV_VAR: "https://some-new-url.{{ requiredEnv "BASE_DOMAIN" }}"
 ```
-FOO=BAR
+
+3. If the value of the new environment variable can't be set dynamically with the environment name or required environment variables (i.e. a true/false feature flag), you must now modify each environment.env file in helmfile/overrides/\<environmnent\>.env (State Value Files) to include your new variable and its configuration for each environment.
+
+staging.env
+
+```yaml
+
+ADMIN_DOCKER_TAG: "1d912b8"
+API_DOCKER_TAG: "1270b81"
+DOCUMENT_DOWNLOAD_DOCKER_TAG: "7f5dc9d"
+......
+GC_ORGANISATIONS_BUCKET_NAME: "notification-canada-ca-staging-gc-organisations"
+SOME_NEW_ENV_VAR: "true"
+
 ```
 
-and then in `kustomization.yaml` add:
+production.env 
 
 ```yaml
-- name: FOO
-  objref:
-    kind: ConfigMap
-    name: application-config
-    apiVersion: v1
-  fieldref:
-    fieldpath: data.FOO
+
+ADMIN_DOCKER_TAG: "1d912b8"
+API_DOCKER_TAG: "1270b81"
+DOCUMENT_DOWNLOAD_DOCKER_TAG: "7f5dc9d"
+......
+GC_ORGANISATIONS_BUCKET_NAME: "notification-canada-ca-production-gc-organisations"
+SOME_NEW_ENV_VAR: "false"
 
 ```
 
-under the `vars:` key.
+### Steps for API
 
-You can now reference the variable in you other manifest files using `$(FOO)`.
+Since the API is run on both kubernetes and AWS Lambda, you will need to modify the Terraform parameter store configuration as well. Follow the steps in the preceeding section and then [update the terraform file as necessary.](https://github.com/cds-snc/notification-terraform/blob/main/aws/lambda-api/secrets_manager.tf)
```

Comment:
because you say "as well" here, does that mean the above section if required for API + this section is required for API? If so we need to make that clear and change this heading for example:
https://github.com/cds-snc/notification-manifests/blob/cbe51b75bbd06ccb7df04b6c64f4b33277e9cd30/README.md#steps-for-everything-but-api

Reply:
Reworded to make it clear you need to both